configs/postfix/main.cf.sample

```diff
@@ -195,24 +241,98 @@ relayhost =
 # This value: 26.214.400 = 25MB
 mailbox_size_limit = 26214400
 
-# The local network interface the mail system receives mail on.
+# The network interface the mail system receives mail on.
 #
 # If only one IP address is provided, this address is also used to actually
 # send outbound mail.
 #
 # Ref: https://www.postfix.org/postconf.5.html#inet_interfaces
 #
-# The documentation suggest to specify the outbound address in ``master.cf``.
+# Accepted values: ``all`` to listen on all interfaces or an IPv4/IPv6 address
+#                  to listen to.
 #
-# FIXME: See https://github.com/Mischback/mailsrv/issues/13#issuecomment-1293650333
-inet_interfaces = all
+# FIXME: See https://github.com/Mischback/mailsrv/issues/19
+# CONFIGURATION_REQUIRED!
+inet_interfaces = ${MAILSRV_POSTFIX_INTERFACES}
+
+# Use this IPv4 address for outgoing connections.
+#
+# Ref: https://www.postfix.org/postconf.5.html#smtp_bind_address
+#
+# In this setup, this is specified globally (in ``main.cf``), but there is an
+# option to use this setting dedicatedly in ``master.cf`` for services. See
+# https://www.postfix.org/postconf.5.html#inet_interfaces for reference.
+#
+# Accepted values: an IPv4 address or [blank]
+#
+# Default value: [blank]
+#
+# CONFIGURATION_REQUIRED!
+smtp_bind_address = ${MAILSRV_BIND_IPV4}
+
+# Use this IPv6 address for outgoing connections.
+#
+# Ref: https://www.postfix.org/postconf.5.html#smtp_bind_address6
+#
+# In this setup, this is specified globally (in ``main.cf``), but there is an
+# option to use this setting dedicatedly in ``master.cf`` for services. See
+# https://www.postfix.org/postconf.5.html#inet_interfaces for reference.
+#
+# Accepted values: An IPv6 address or [blank]
+#
+# Default value: [blank]
+#
+# CONFIGURATION_REQUIRED!
+smtp_bind_address6 = ${MAILSRV_BIND_IPV6}
+
+# Force the address bind as specified by smtp_bind_address[6]
+#
+# Actually the binding should be enforced. This logs lots of warnings while
+# Postfix starts up, if ``smtp_bind_address`` and ``smtp_bind_address6`` are
+# not provided and this is set to ``yes``. If your setup uses either of the
+# mentioned configuration values, consider turning this to ``yes``.
+#
+# Ref: https://www.postfix.org/postconf.5.html#smtp_bind_address_enforce
+#
+# Default value: no
+#
+# CONFIGURATION_REQUIRED!
+smtp_bind_address_enforce = no
```

Comment:
It should be mentioned, that this raises lots of warnings during Postfix's startup, if neither ``smtp_bind_address`` nor ``smtp_bind_address6`` is specified or empty!